src/test/java/eu/europa/ted/efx/EfxExpressionTranslatorTest.java

```diff
@@ -1162,4 +1162,175 @@ void testTimeFromStringFunction() {
     assertEquals("xs:time(PathNode/TextField/normalize-space(text()))",
         test("ND-Root", "time(BT-00-Text)"));
   }
+
+  /*** Sequence Functions ***/
+
+  @Test
+  void testDistinctValuesFunction_WithStringSequences() {
+    assertEquals("distinct-values(('one','two','one'))",
+        test("ND-Root", "distinct-values(('one', 'two', 'one'))"));
+  }
+
+  @Test
+  void testDistinctValuesFunction_WithNumberSequences() {
+    assertEquals("distinct-values((1,2,3,2,3,4))",
+        test("ND-Root", "distinct-values((1, 2, 3, 2, 3, 4))"));
+  }
+
+  @Test
+  void testDistinctValuesFunction_WithDateSequences() {
+    assertEquals("distinct-values((xs:date('2018-01-01Z'),xs:date('2020-01-01Z'),xs:date('2018-01-01Z'),xs:date('2022-01-02Z')))",
+        test("ND-Root", "distinct-values((2018-01-01Z, 2020-01-01Z, 2018-01-01Z, 2022-01-02Z))"));
+  }
+
+  @Test
+  void testDistinctValuesFunction_WithTimeSequences() {
+    assertEquals("distinct-values((xs:time('12:00:00Z'),xs:time('13:00:00Z'),xs:time('12:00:00Z'),xs:time('14:00:00Z')))",
+        test("ND-Root", "distinct-values((12:00:00Z, 13:00:00Z, 12:00:00Z, 14:00:00Z))"));
+  }
+
+  @Test
+  void testDistinctValuesFunction_WithBooleanSequences() {
+    assertEquals("distinct-values((true(),false(),false(),false()))",
+        test("ND-Root", "distinct-values((TRUE, FALSE, FALSE, NEVER))"));
+  }
+  
+  @Test
+  void testDistinctValuesFunction_WithFieldReferences() {
+    assertEquals("distinct-values(PathNode/TextField)",
+        test("ND-Root", "distinct-values(BT-00-Text)"));
+  }
+
+  /* Union */
+
+  @Test
+  void testUnionFunction_WithStringSequences() {
+    assertEquals("distinct-values(('one','two'), ('two','three','four'))",
```

Comment:
Correct expected XPath is:
`distinct-values((('one','two'), ('two','three','four')))`

Reply:
Fixed